config/initializers/load_pre_production_formats.rb

```diff
@@ -3,7 +3,7 @@ def pre_production
   Dir["lib/documents/schemas/*.json"].each do |file|
     read_file = File.read(file)
     parsed_file = JSON.parse(read_file)
-    if parsed_file["pre_production"]
+    if parsed_file["pre_production"] && parsed_file["name"] != "Research for Development Outputs"
```

Comment:
The name is currently volatile due to iteration / liaison with content designers. Can we go by the `base_path` or `content_id` instead?